File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.java
class Solution {
    public int[] solution(String myString) {
        
        int temp = 0;
        String o[] = myString.split("x",-1);    // "x"를 기준으로 문자열을 나눔
        
        int[] answer = new int[(o.length)];
            
        for(int i=0; i<answer.length; i++){
            temp = o[i].length();               // length()로 문자열의 길이를 구함
            answer[i] += temp;                  // 문자열의 길이를 answer에 넣음
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.js
function solution(myString) {
    var answer = [];
    var str = myString.split("x");      // 문자열을 x를 기준으로 나눔
    var o = 0;
    
    for(let i of str){                  // i는 배열의 값
        if(i.length>0)
            answer[o] = i.length;
        else
           answer[o] = 0; 
        o++;
    }
    
    return answer;
}